### PR TITLE
Add Polly 0.2.2 bindings for the Linux epoll system call

### PR DESCRIPTION
**Pull request by lindig**
```
opam-version: "2.0"
name: "polly"
synopsis: "Bindings for the Linux epoll system call"
description: """
Bindings for the Linux epoll system call. The binding avoids
  most allocation in the event loop by iterating over all file
  descriptios that are reported as ready ."""
maintainer: "Christian Lindig <christian.lindig@citrix.com>"
authors: "Christian Lindig <christian.lindig@citrix.com>"
license: "MIT"
homepage: "https://github.com/lindig/polly"
doc: "https://github.com/lindig/polly"
bug-reports: "https://github.com/lindig/polly/issues"
depends: [
  "dune" {build}
  "ocaml"
  "cmdliner"
  "base-unix"
  "conf-linux-libc-dev"
]
build: ["dune" "build" "-p" name "-j" jobs "@install"]
dev-repo: "git://github.com/lindig/polly.git"

url {
  src: "https://github.com/lindig/polly/archive/0.2.2/polly-0.2.2.tar.gz"
  checksum: "md5=a25291f7adb6b1d7bdd7513f2c22d9de"
}
```

### CODE REVIEW

File: packages/polly/polly.0.2.2/opam [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  most allocation in the event loop by iterating over all file
+  descriptios that are reported as ready ."""
+maintainer: "Christian Lindig <christian.lindig@citrix.com>"
+authors: "Christian Lindig <christian.lindig@citrix.com>"
+license: "MIT"
+homepage: "https://github.com/lindig/polly"
+doc: "https://github.com/lindig/polly"
+bug-reports: "https://github.com/lindig/polly/issues"
+depends: [
+  "dune" {build}
```
Comment (mseri) [inline packages/polly/polly.0.2.2/opam:14]:
```suggestion
  "dune" {>= "2.0"}
```

dune should not be tagged with the build tag due to forward compatibility issues, according to the dune-project file you need at least 2.0

Comment (mseri) [reply packages/polly/polly.0.2.2/opam]: @lindig can you upstream this change?

Comment (lindig) [reply packages/polly/polly.0.2.2/opam]: I will change it here and then change the opam file in the source repository to match it.

File: packages/polly/polly.0.2.2/opam [hunk 2]
```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+synopsis: "Bindings for the Linux epoll system call"
+description: """
+Bindings for the Linux epoll system call. The binding avoids
+  most allocation in the event loop by iterating over all file
+  descriptios that are reported as ready ."""
```
Comment (avsm) [inline packages/polly/polly.0.2.2/opam:6]:
```suggestion
  descriptiors that are reported as ready ."""
```

**Comment by mseri** [comment]
I don’t know what is wrong with camelus. For the rest seems good to go

**Pull request merged**